main.go

```diff
@@ -147,20 +161,29 @@ func cleanWindows(p string) string {
 func main() {
 
 	var headers headerList
+	var proxyURL url.URL
 
 	outDir := flag.String("output", "", "Source file output directory - REQUIRED")
-	url := flag.String("url", "", "URL or path to the Sourcemap file - REQUIRED")
+	urlflag := flag.String("url", "", "URL or path to the Sourcemap file - REQUIRED")
+	proxy := flag.String("proxy", "", "Explicity HTTP client proxy")
```

Comment:
`Explicitly HTTP client proxy` is not good English, "Proxy URL" instead perhaps?